diff --git a/vcx/ci/Jenkinsfile b/vcx/ci/Jenkinsfile
index c36c9b2..e5664b5 100644
--- a/vcx/ci/Jenkinsfile
+++ b/vcx/ci/Jenkinsfile
@@ -112,15 +112,15 @@ def mainUbuntu() {
                     withCredentials([usernameColonPassword(credentialsId: 'jenkins-kraken-svc', variable: 'KRAKEN_CREDENTIALS')]) {
                         libvcx.inside("--network=host") {
                             // ******** Publish Libvcx Debian to repo ********
-                            sh "vcx/ci/scripts/publish.sh \"${KRAKEN_CREDENTIALS}\" \"libvcx*.deb\" https://kraken.corp.evernym.com/repo/portal_dev/upload"
-                            sh "vcx/ci/scripts/publish.sh \"${KRAKEN_CREDENTIALS}\" \"libvcx*.deb\" https://kraken.corp.evernym.com/repo/agency_dev/upload"
+                            sh "vcx/ci/scripts/publish.sh \"libvcx*.deb\" https://kraken.corp.evernym.com/repo/portal_dev/upload"
+                            sh "vcx/ci/scripts/publish.sh \"libvcx*.deb\" https://kraken.corp.evernym.com/repo/agency_dev/upload"
 
                             // ******** Publish Npm Package to filely repo ********
-                            sh "vcx/ci/scripts/publish.sh \"${KRAKEN_CREDENTIALS}\" \"*.tgz\" https://kraken.corp.evernym.com/repo/npm/upload"
+                            sh "vcx/ci/scripts/publish.sh \"*.tgz\" https://kraken.corp.evernym.com/repo/npm/upload"
 
                             // ******** Publish vcx Debian To Repo (NPM PACKAGES)
-                            sh "vcx/ci/scripts/publish.sh \"${KRAKEN_CREDENTIALS}\" \"vcx_*.deb\" https://kraken.corp.evernym.com/repo/agency_dev/upload"
-                            sh "vcx/ci/scripts/publish.sh \"${KRAKEN_CREDENTIALS}\" \"vcx_*.deb\" https://kraken.corp.evernym.com/repo/portal_dev/upload"
+                            sh "vcx/ci/scripts/publish.sh \"vcx_*.deb\" https://kraken.corp.evernym.com/repo/agency_dev/upload"
+                            sh "vcx/ci/scripts/publish.sh \"vcx_*.deb\" https://kraken.corp.evernym.com/repo/portal_dev/upload"
 
 
                             // ******** Archive with Jenkins ********
@@ -130,7 +130,7 @@ def mainUbuntu() {
                             java.inside() {
                                 sh 'cp $settingsFile .'
                                 sh "chmod +x vcx/wrappers/java/ci/publishJar.sh"
-                                sh 'vcx/wrappers/java/ci/publishJar.sh'
+                                sh 'vcx/wrappers/java/ci/publishJar.sh'find
                             }
                         }
 
@@ -231,7 +231,7 @@ def fail() {
 
 def buildAndroid(arch) {
     node('ubuntu') {
-        try {
+        try {\"${KRAKEN_CREDENTIALS}\
             checkout scm
             android = build('android', 'vcx/ci/android.dockerfile', 'vcx')
 
diff --git a/vcx/ci/scripts/publish.sh b/vcx/ci/scripts/publish.sh
index ef363ae..35820fd 100755
--- a/vcx/ci/scripts/publish.sh
+++ b/vcx/ci/scripts/publish.sh
@@ -5,9 +5,9 @@ if [ $# -ne 3 ]; then
     exit 1
 fi
 
-CREDENTIALS=$1
-FILENAME=$2
-URL=$3
+CREDENTIALS=$KRAKEN_CREDENTIALS
+FILENAME=$1
+URL=$2
 LOOKUP_DIR="output"
 
 echo "Filename: ${FILENAME}"
